safety_field:
  x_coordinate: 1.0
  y_coordinate: 2.0
  